Snabble/UI/ShopFinder/Model/OpeningHourViewModel.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  OpeningHourViewModel.swift
+//  Snabble
+//
+//  Created by Uwe Tilemann on 25.08.22.
+//
+
+import Foundation
+import SwiftUI
+
+public protocol OpeningHourProvider {
+    var day: String? { get }
+    var hour: String? { get }
+}
+
+extension Asset {
+    public static func localizedString(forKey key: String, arguments: CVarArg..., table: String? = nil, value: String? = nil, fallback: String? = nil, domain: Any? = domain) -> String {
+        let string = localizedString(forKey: key, arguments: arguments, table: table, value: value, domain: domain)
+        
+        if string == key, fallback != nil {
+            return fallback!
+        }
+        return string
+    }
+}
```

Comment:
Kannst du mir erklären wofür du es brauchst? Kann es eventuell sein das die Implementierung in `AppDelegate+AssetProviding` das Problem löst oder du das benötigst da du einen Key aus der Sample App anfragst? `Sample.Shop.Detail.hour`

Reply:
Das habe ich mir gemacht, weil ich folgenden Code nicht schreiben wollte:
```
let key = "Snabble.shop.Detail.hour"
var hourText = Asset.localizedString(forKey: key)
if hourText == key {
    hourText = ""
}
```

Reply:
Der Code sollte nicht notwendig sein. Der Key `Snabble.Shop.Detail.hour` ist im SDK vorhanden.

Reply:
was ist, wenn dort für eine Sprache keine Übersetzung dran ist ""?

Reply:
Standard Übersetzung ist immer English. Wir müssen deutsch und englisch immer hinterlegen.

Reply:
3c0a81da1f283efcf9cb4eb9269fe554686c43a9